<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SCAN_ScanResultWidget</class>
 <widget class="QWidget" name="SCAN_ScanResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="margin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splHorizontal">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Result List</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbAddScan">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Add scan</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../db/src/gui/db.qrc">
             <normaloff>:/images/icons/add.png</normaloff>:/images/icons/add.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDeleteScan">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Delete scan</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/images/icons/delete.png</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbSaveScan">
           <property name="toolTip">
            <string>Save scan</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../db/src/gui/db.qrc">
             <normaloff>:/images/icons/database_save.png</normaloff>:/images/icons/database_save.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbRevertScan">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Undo current changes</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../db/src/gui/db.qrc">
             <normaloff>:/images/icons/arrow_undo.png</normaloff>:/images/icons/arrow_undo.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableView" name="tvScan">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Result Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leResultName"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Scan Type</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QComboBox" name="cbScanType"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbSelectScanType">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../db/src/gui/db.qrc">
               <normaloff>:/images/icons/wrench.png</normaloff>:/images/icons/wrench.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Person</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <widget class="QComboBox" name="cbPerson"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbSelectPerson">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../db/src/gui/db.qrc">
               <normaloff>:/images/icons/wrench.png</normaloff>:/images/icons/wrench.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab1">
          <attribute name="title">
           <string>Answer Result</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="margin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <widget class="QSplitter" name="splVertical">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QGroupBox" name="gbQuestions">
              <property name="title">
               <string>Questions</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
               <property name="leftMargin">
                <number>3</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="rightMargin">
                <number>3</number>
               </property>
               <property name="bottomMargin">
                <number>3</number>
               </property>
               <property name="horizontalSpacing">
                <number>3</number>
               </property>
               <property name="verticalSpacing">
                <number>1</number>
               </property>
               <item row="0" column="0" colspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>1</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="pbAddQuestion">
                   <property name="minimumSize">
                    <size>
                     <width>24</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>24</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>Add questions</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="../../../db/src/gui/db.qrc">
                     <normaloff>:/images/icons/paintcan.png</normaloff>:/images/icons/paintcan.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pbDeleteQuestion">
                   <property name="minimumSize">
                    <size>
                     <width>24</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>24</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="toolTip">
                    <string>Delete questions</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normalon>:/images/icons/delete.png</normalon>
                    </iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Click row to select</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QTableView" name="tvQuestion">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>4</horstretch>
                   <verstretch>1</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="layoutWidget_1">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="Line" name="line">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="gbAnswers">
                 <property name="title">
                  <string>Answers</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0">
                  <property name="leftMargin">
                   <number>3</number>
                  </property>
                  <property name="topMargin">
                   <number>1</number>
                  </property>
                  <property name="rightMargin">
                   <number>3</number>
                  </property>
                  <property name="bottomMargin">
                   <number>3</number>
                  </property>
                  <property name="horizontalSpacing">
                   <number>3</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>1</number>
                  </property>
                  <item row="0" column="0" colspan="2">
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <property name="spacing">
                     <number>1</number>
                    </property>
                    <item>
                     <spacer name="horizontalSpacer_4">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>Click row to select</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <widget class="QTableView" name="tvAnswer">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                      <horstretch>0</horstretch>
                      <verstretch>4</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="toolTip">
                       <string>Sequence Number</string>
                      </property>
                      <property name="text">
                       <string>Number</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leAnswerNumber">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>75</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>75</width>
                        <height>16777215</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Value</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leValue">
                      <property name="minimumSize">
                       <size>
                        <width>75</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>75</width>
                        <height>16777215</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_7">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab2">
          <attribute name="title">
           <string>Score Result</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="horizontalSpacing">
            <number>3</number>
           </property>
           <property name="verticalSpacing">
            <number>1</number>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pbRefreshScore">
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Refresh score result</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../../db/src/gui/db.qrc">
                 <normaloff>:/images/icons/arrow_refresh.png</normaloff>:/images/icons/arrow_refresh.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QWebView" name="wvScoreResult">
             <property name="url">
              <url>
               <string>about:blank</string>
              </url>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab3">
          <attribute name="title">
           <string>Report</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="margin">
            <number>3</number>
           </property>
           <property name="spacing">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="pbRefreshReport">
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Refresh report</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../../db/src/gui/db.qrc">
                 <normaloff>:/images/icons/arrow_refresh.png</normaloff>:/images/icons/arrow_refresh.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QWebView" name="wvReport">
             <property name="url">
              <url>
               <string>about:blank</string>
              </url>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../db/src/gui/db.qrc"/>
 </resources>
 <connections/>
</ui>
